src/core/site.jl

```diff
@@ -10,17 +10,20 @@ Inputs related to the physical location:
     roof_squarefeet::Union{Real, Nothing} = nothing,
     min_resil_time_steps::Int=0, # The minimum number consecutive timesteps that load must be fully met once an outage begins. Only applies to multiple outage modeling using inputs outage_start_time_steps and outage_durations.
     mg_tech_sizes_equal_grid_sizes::Bool = true,
-    node::Int = 1,
+    sector::String = "commercial/industrial",
+    federal_sector_state::String = "",
+    federal_procurement_type::String = "",
     CO2_emissions_reduction_min_fraction::Union{Float64, Nothing} = nothing,
     CO2_emissions_reduction_max_fraction::Union{Float64, Nothing} = nothing,
     bau_emissions_lb_CO2_per_year::Union{Float64, Nothing} = nothing, # Auto-populated based on BAU run. This input will be overwritten if the BAU scenario is run, but can be user-provided if no BAU scenario is run.
     bau_grid_emissions_lb_CO2_per_year::Union{Float64, Nothing} = nothing,
-    renewable_electricity_min_fraction::Real = 0.0,
+    renewable_electricity_min_fraction::Union{Float64, Nothing} = nothing,
     renewable_electricity_max_fraction::Union{Float64, Nothing} = nothing,
     include_grid_renewable_fraction_in_RE_constraints::Bool = false,
     include_exported_elec_emissions_in_total::Bool = true,
     include_exported_renewable_electricity_in_total::Bool = true,
     outdoor_air_temperature_degF::Union{Nothing, Array{<:Real,1}} = nothing,
+    node::Int = 1,
```

Comment:
I know you already have a TODO to add more documentation, but just noting that here would be a good place for a !!! note and I'd suggest including there a comment that the defaults *do not* override a user-provided value.